client/src/components/Post/Post.tsx

```diff
@@ -5,12 +5,14 @@ import {TrendzButton} from "../common/TrendzButton/TrendzButton";
 import {parseJwt} from "../Routing/utils";
 import Modal from "react-modal";
 import {deletePost, getPostData} from "../../api/PostApi";
-import {MdThumbDown, MdThumbUp} from 'react-icons/md';
+import {MdThumbDown, MdThumbUp, MdDelete, MdModeEdit} from 'react-icons/md';
 import {PostType, TopicType} from "../types/types";
 import {Formik,} from 'formik';
 import * as yup from 'yup';
-import {createComment} from "../../api/CommentApi";
+import TimeAgo from 'react-timeago'
```

Comment:
@Agusdelavega this needs imports in package.json, the same way you do it w/ mvn in the backend. It does work fine, but you probably did a local npm install instead of adding the correct types, for example:
`    "@types/react-timeago": "4.1.1",
      "react-timeago": "5.2.0",
`